ansible_collections/arista/avd/molecule/eos_designs_unit_tests/documentation/devices/DC1-BL2A.md

```diff
@@ -138,7 +138,7 @@ ntp server vrf MGMT 192.168.200.5 prefer
 ```eos
 !
 management api http-commands
-   protocol https
+   no default-services
```

Comment:
The changes here looks wrong. Please take another look. I will try to identify what caused this in the template.